src/app/js/dataset/Pagination.js

```diff
@@ -0,0 +1,182 @@
+// Taken from https://github.com/ENDiGo/pagination-material-ui
+
+import React, { Component, PropTypes } from 'react';
+import IconButton from 'material-ui/IconButton';
+import MenuItem from 'material-ui/MenuItem';
+import SelectField from 'material-ui/SelectField';
+
+import ChevronLeft from 'material-ui/svg-icons/navigation/chevron-left';
+import ChevronRight from 'material-ui/svg-icons/navigation/chevron-right';
+
+import translate from 'redux-polyglot/translate';
+import { polyglot as polyglotPropTypes } from '../lib/propTypes';
+
+const styles = {
+    pagination: {
+        borderTop: '1px solid rgb(224, 224, 224)',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'flex-end',
+    },
+    elements: {
+        display: 'flex',
+        alignItems: 'center',
+        height: 56,
+        marginLeft: 16,
+    },
+    pageSelect: {
+        marginLeft: 0,
+    },
+    label: {
+        color: '#999',
+        fontWeight: 300,
+        fontSize: 12,
+    },
+    select: {
+        width: 100,
+        textAlign: 'right',
+    },
+    underline: {
+        display: 'none',
+    },
+};
+
+class Pagination extends Component {
+    static propTypes = {
+        onChange: PropTypes.func.isRequired,
+        p: polyglotPropTypes,
+        perPage: PropTypes.number,
+        total: PropTypes.number.isRequired,
+    };
+
+    static defaultProps = {
+        total: 0,
+        perPage: 10,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentPage: 1,
```

Comment:
Because later, this will probably be an external Pagination component from another package. It should be already but unfortunately, the package has not been compiled correctly.